source/common/common/utility.h

```diff
@@ -64,6 +64,26 @@ class ProdMonotonicTimeSource : public MonotonicTimeSource {
   static ProdMonotonicTimeSource instance_;
 };
 
+/**
+ * Class used for creating non-copying std::istream's. See IMemoryStream below.
+ */
+class MemoryStreamBuffer : public std::streambuf {
```

Comment:
nit: Can we call this `ConstMemoryStreamBuffer` ?

Reply:
Renamed.